en/08_Changelogs/6.1.0.md

```diff
@@ -133,11 +133,19 @@ A number of new indexes are added as part of this release. Along with the [compo
 
 Note that building indexes for large tables may result in a noticable increase in the time it takes to run `sake db:build` the first time after upgrading your project.
 
-### Non-blocking file-based sessions {#non-blocking-sessions}
+### Non-blocking session handlers {#non-blocking-sessions}
 
 The default file-based session handler for PHP holds a lock on the session file while the session is open. This means that multiple concurrent requests from the same user have to wait for one another to finish processing after a session has been started. This includes AJAX requests.
 
-To resolve this problem, a new [`Session.save_handler`](api:SilverStripe\Control\Session->save_handler) configuration property has been added. By default this is set to [`FileSessionHandler`](api:SilverStripe\Control\SessionHandler\FileSessionHandler) which is *non-blocking* which means that multiple concurrent requests from the same user don't have to wait for one another to finish.
+To resolve this problem, we have provided three new session save handlers. Each of these are *non-blocking* which means that multiple concurrent requests from the same user don't have to wait for one another to finish.
```

Comment:
Do we need a link / instructions for how to keep sessions blocking in the changelog in case the non-blocking sessions break something?

Reply:
That's already in the changelog - set the thingy to null.